src/app/portlets/dot-edit-page/layout/components/dot-theme-selector/dot-theme-selector.component.ts

```diff
@@ -39,7 +43,20 @@ export class DotThemeSelectorComponent implements OnInit {
                 'dot.common.apply',
                 'dot.common.cancel'
             ])
-            .subscribe();
+            .subscribe(() => {
+                this.closeDialogAction = {
+                    label: this.dotMessageService.get('dot.common.cancel'),
+                    action: () => {}
+                };
+                this.applyDialogAction  = {
+                    label: this.dotMessageService.get('dot.common.apply'),
+                    disabled: true,
+                    action: () => {
+                        this.apply();
+                    }
+                };
+                this.dotDialog.reRecenter();
```

Comment:
Could we do this when the dialog is open? I mean when the show attribute is set to true